src/java/generator/UnitGenerator.java

```diff
@@ -0,0 +1,52 @@
+package generator;
+
+import map.BinaryMask;
+import map.Unit;
+import map.SCMap;
+import util.Vector2f;
+import util.Vector3f;
+
+import java.util.Random;
+
+public strictfp class UnitGenerator {
+    public final String[] STRUCTURES = {
+
+    };
```

Comment:
is that used?

Reply:
This is not used in this iteration however I left it in since this will be the place that civilian structures are added so they can be placed easily.